src/main/java/com/gregtechceu/gtceu/data/lang/LangHandler.java

```diff
@@ -1257,6 +1034,10 @@ public static void init(RegistrateLangProvider provider) {
 
         provider.add("gtceu.gui.machinemode.title", "Active Machine Mode");
         provider.add("gtceu.gui.machinemode", "Active Machine Mode: %s");
+        provider.add("gtceu.machine.available_recipe_map_1.tooltip", "Available Recipe Types: %s");
+        provider.add("gtceu.machine.available_recipe_map_2.tooltip", "Available Recipe Types: %s, %s");
+        provider.add("gtceu.machine.available_recipe_map_3.tooltip", "Available Recipe Types: %s, %s, %s");
+        provider.add("gtceu.machine.available_recipe_map_4.tooltip", "Available Recipe Types: %s, %s, %s, %s");
```

Comment:
Wouldn't separate entries for "Available Recipe Types: %s" and for the separator be better here?

Reply:
..I didn't do that.

Reply:
also, and what?

Reply:
Oops, typing skill issue lol - should've been
> and `", "` for the separator

Though if you only moved stuff around here, let's just leave it for now